File: A/buttons.c
#include <stdio.h>

void solve() {
    int a, b, c;
    scanf("%d %d %d", &a, &b, &c);
    int first = a + (c + 1) / 2;
    int second = b + c / 2;
    puts((first > second ? "First" : "Second"));
}

int main(void) {
    int t;
    scanf("%d", &t);
    while (t--) {
        solve();
    }
    return 0;
}

File: A/buttons.py
import sys
input = sys.stdin.readline

def solve():
    a, b, c = map(int, input().split())
    first, second = a + (c + 1) // 2, b + c // 2
    print("First" if first > second else "Second")

for _ in range(int(input())):
    solve()
